std/range/package.d

```diff
@@ -5399,7 +5407,7 @@ if (isIntegral!(CommonType!(B, E)) || isPointer!(CommonType!(B, E)))
         @property inout(Value) front() inout { assert(!empty); return current; }
         void popFront() { assert(!empty); ++current; }
 
-        @property inout(Value) back() inout { assert(!empty); return cast(inout(Value))(pastLast - 1); }
+        @property inout(Value) back() inout{ assert(!empty); return cast(inout(Value))(pastLast - 1); }
```

Comment:
It churns the git blame and if we don't pay attention to avoid unrelated changes, git blame becomes useless. Also it makes things a lot harder for people who skim over the diffs.